src/cat.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Direct URL to the image
+API_URL="https://storage.googleapis.com/readbee-public-dev/flower-kitten.gif"
+
+# Specify the path and filename for saving the image
+IMAGE_FILE="/path/to/save/cat_image.gif" # Change extension to .gif to match the image type
```

Comment:
Clearly this is a copy-pasta from Google - you should at least select a reasonable path to replace this with.